packages/backend/tsconfig.index.json

```diff
@@ -0,0 +1,20 @@
+{
+    // This file will extend the tsconfig.json file for the test environment
+    // This means we will not apply the noUncheckedIndexedAccess option to the test files
+    // Change build method in package.json to use this file
+    // It is temporary disabled in package.json until we fix all the errors
```

Comment:
IN order to permanently change this file in package.json, we'll have to modify also the dockerfile, since that file is `COPY`ed separately.

Reply:
unfortunately, I don't think  there is any way to partilaly activate this rule (or just warn) . So until we fix all errors, we can't use it.